### PR TITLE
Skeleton version of backend structure. We will be building on top of this. 

### PR DESCRIPTION
**Pull request by F-Gulmez**

### CODE REVIEW

File: api/.gitignore
Comment (Seyi-Toluhi) [inline api/.gitignore:1]: can we get the .DS_Store inside the gitignore.

File: api/tests/lib/token.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { generateToken, decodeToken } = require("../../lib/token");
+
+describe("TokenGenerator", () => {
+  describe("jsonwebtoken", () => {
+    test("returns a token containing user_id that is valid for 10 minutes", () => {
```
Comment (tomgame984) [inline api/tests/lib/token.test.js]: Test wording to be amend to 30 minutes

File: api/tests/lib/token.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      expect(token_1).not.toEqual(token_2);
+
+      // Decode tokens
+      const payload_1 = decodeToken(token_1);
+      const payload_2 = decodeToken(token_2);
+
+      expect(payload_1.user_id).toEqual(id_1);
+      expect(payload_2.user_id).toEqual(id_2);
+
+      // Token is valid for 600 seconds (30 minutes)
```
Comment (tomgame984) [inline api/tests/lib/token.test.js]: Change "Token is valid for 1800 seconds".

**Review by tomgame984 — CHANGES_REQUESTED** [review]
Test cases reviewed and OK as baseline tests.  Further tests likely to be required as app develops.  
Please make amendments as per comments and advise when done.

**Review by F-Gulmez — COMMENTED** [review]
Made the changes asked and committed in the branch

**Review by F-Gulmez — COMMENTED** [review]
Implemented requested changes

**Review by F-Gulmez — COMMENTED** [review]
Done

**Review by tomgame984 — DISMISSED** [review]

**Review by tomgame984 — APPROVED** [review]

**Review by Seyi-Toluhi — APPROVED** [review]

**Review by JackCollett — APPROVED** [review]

**Pull request merged**